django/test/utils.py

```diff
@@ -200,6 +201,9 @@ def setup_databases(verbosity, interactive, keepdb=False, debug_sql=False, paral
         for alias in connections:
             connections[alias].force_debug_cursor = True
 
+    # Now run the system checks. They aren't run until now since some checks require database access.
+    call_command('check', verbosity=verbosity)
```

Comment:
I'm not sure if this is the best place for this -- doesn't seem like "running system checks" is related to "setting up databases". Anyway, this function is documented in topics/testing/advanced.txt and the docs should be updated if this change happens.